user-guide/components/SubReport/creating-master-detail-report.md

```diff
@@ -16,30 +16,14 @@ img[alt$="><"] {
 
 </style>
 
-# SubReport
+# Creating Master-Detail Reports with SubReports
 
-A SubReport is a report component that allows you to embed one report inside another report. This lets you create complex report layouts, display hierarchical data relationships, and build reusable report components that can be shared across multiple reports.
+Using SubReports in the Web Report Designer allows you to embed reports within other reports for creating master-detail hierarchies and reusable report components.
 
-SubReports act as containers that automatically adjust their size based on the content of the embedded report. You can use SubReports to create:
-
-* Master-detail relationships&mdash;Display related data in a parent-child format (for example, customers and their orders).
-* Reusable components&mdash;Create shared headers, footers, or report sections that can be used across multiple reports.
-* Complex layouts&mdash;Combine different data sources and report structures within a single report.
-* Hierarchical data&mdash;Show nested data relationships with proper grouping and organization.
-
-The next video demonstrates how to [create master-detail](#creating-master-detail-reports-with-subreports) reports by using SubReports in the Telerik Web Report Designer:
+The next video demonstrates how to create master-detail reports by using SubReports in the Telerik Web Report Designer following the below steps:
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/GnZi9PP9EK8?si=qaBfIZnbUu1fQzyK&rel=0" title="Adding a SubReport Item in the Telerik Web Report Designer" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
I suggest moving the video and sentence that introduces it right before step 1. This way, the video will not interrupt the flow of the article (1. Introductory sentence, 2. Description of the tutorial 3. Video that demonstrates the tutorial).